int-set/Cargo.toml

```diff
@@ -7,6 +7,7 @@ description = "A fast sparse and invertible bit set for u32's. Port of harfbuzz'
 
 [dependencies]
 font-types = { version = "0.5.5", path = "../font-types"}
+thiserror = "1.0.58"
```

Comment:
I think we don't use this in fontations and probably shouldn't here either, just implement the error manually